pages/search/index.js

```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+import GameSearchedList from "@/components/GameSearchedList";
+
+import { useState } from "react";
+import useSWR from "swr";
+
+export default function SearchPage() {
+  const { data } = useSWR("./api/games");
+  const [searchInput, setSearchInput] = useState("");
+
+  const searchedGames =
+    searchInput.length > 0
+      ? data.filter((game) =>
+          `${game.name}`.includes(searchInput.toLowerCase())
+        )
+      : [];
+
+  return (
+    <>
+      <Input
+        type="text"
+        id="text"
+        placeholder="Insert name"
+        value={searchInput}
+        onChange={(event) => setSearchInput(event.target.value)}
+      />
+      <GameSearchedList data={searchedGames} />
```

Comment:
I would call the component `GameList`, and I would call the prop `games` ^^

Reply:
Thank you for review Diego,
we have already our GameList component in the main branch, and we created GameSearchedList because we have different orientation of presenting the games on different pages, so we've chosen this variant